Mathlib/NumberTheory/ArithmeticFunction.lean

```diff
@@ -196,41 +207,46 @@ section AddMonoid
 
 variable [AddMonoid R]
 
-instance : Add (ArithmeticFunction R) :=
+instance add : Add (ArithmeticFunction R) :=
   ⟨fun f g => ⟨fun n => f n + g n, by simp⟩⟩
 
 @[simp]
 theorem add_apply {f g : ArithmeticFunction R} {n : ℕ} : (f + g) n = f n + g n :=
   rfl
 #align nat.arithmetic_function.add_apply Nat.ArithmeticFunction.add_apply
 
-instance : AddMonoid (ArithmeticFunction R) :=
-  { ArithmeticFunction.hasZero R,
-    ArithmeticFunction.hasAdd with
+instance instAddMonoid : AddMonoid (ArithmeticFunction R) :=
+  { ArithmeticFunction.zero R,
+    ArithmeticFunction.add with
     add_assoc := fun _ _ _ => ext fun _ => add_assoc _ _ _
     zero_add := fun _ => ext fun _ => zero_add _
     add_zero := fun _ => ext fun _ => add_zero _ }
+-- porting note: have I aligned this correctly?
```

Comment:
TODO (possibly, after port): do we have this instance for `ZeroHom`? Should we reuse it here? Also, we have `Function.Injective.addMonoid`.